tock/tock/templates/hours/timecard_form.html

```diff
@@ -10,6 +10,26 @@
     </p>
   </div>
 </div>
+{% if float_data.tasks %}
+  <div class="usa-alert usa-alert-error">
+    <div class="usa-alert-body">
+      <p class="usa-alert-text">
+        <b>
+          For {{ float_data.metadata.float_period_start }} to
+          {{ float_data.metadata.float_period_end }} you were scheduled in Float
+          for:
+        </b>
+      </p>
+    <ul>
+      {% for i in float_data.tasks %}
+       <li>-  {{ i.hours_wk }} hours on {{ i.project_name }}
```

Comment:
Will have a look. I agree on the `<li>`, but maybe there was some USWDS weirdness?